_hacks/33.md

```diff
@@ -0,0 +1,5 @@
+---
+hack_number: 33
+date: 2023-01-28
+upcoming: true
```

Comment:
I dont actually know what our automation does if this already exists on December 1...but I guess we can find out!